README.md

```diff
@@ -1,18 +1,61 @@
 # FXLS89xx_Arduino
 This library enables the developer to evaluate NXP FXLS89xx with Arduino.
 
-## Caution when using library
-Make sure that **J6-2 MUST NOT be connected** when you use this library.  
-Since the IOREF pin on Arduino is supplied by 5V, **shorting IOREF (5V) and +3V3 (3.3V) can cause either FXLS89xx board or Arduino board damaged.**  
-This driver offers all functions by only I2C communication because all Arduino output H signal is 5V.
-<img src="J6-2.JPG" width="320px" alt="Cut J6-2 in order NOT to short 5V and 3.3V"/>
+> **Warning**  
+> _**Caution when using library:**_  
+> Make sure that **J6-2 MUST NOT be connected** when you use this library.  
+> Since the IOREF pin on Arduino is supplied by 5V, **shorting IOREF (5V) and +3V3 (3.3V) can cause either FXLS89xx board or Arduino board damaged.**  
```

Comment:
This section modified as a block of warning. It may simplify structure of this document. 
The block of warning is using Github new feature: https://c-a-p-engineer.github.io/tech/2022/09/14/github-markdown-note-warning/